### ISSUE TITLE
Markup conversion, multilines supports

### ISSUE THREAD

**Issue by Nyconing** [butler1233/stackoverflow-chatbot#33]
Markup dual way conversion.
Multilines supports.

scene：
https://chat.stackoverflow.com/transcript/message/51009909#51009909

**Comment by butler1233** [comment]
Yeah that's a helpful "feature" of Stack dropping all formatting on multi line messages. Not really sure how to get around that one.
